docs/conf.py

```diff
@@ -70,6 +70,9 @@
 }
 html_theme = "sunpy"
 html_static_path = [str(_sunpy_static_path), "_static"]
+html_js_files = [
```

Comment:
ha, I also committed a fix for this at the same time in `__init__.py`, which I think is the better place.